clang/include/clang/Driver/Driver.h

```diff
@@ -602,6 +602,17 @@ class Driver {
   // FIXME: This should be in CompilationInfo.
   std::string GetProgramPath(StringRef Name, const ToolChain &TC) const;
 
+  /// GetStdModuleManifestPath - Lookup the path to the Standard library module
```

Comment:
Remove `GetStdModuleManifestPath - ` for new code per https://llvm.org/docs/CodingStandards.html# "Don’t duplicate function or class name at the beginning of the comment."